.github/workflows/extended-ci-longevity-large-partition-asymmetric-cluster-3h-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Build scylla-bench docker image with gocql PR
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled]
+
+jobs:
+  trigger-longevity-large-partition-asymmetric-cluster-3h-test:
+    if: contains(github.event.pull_request.labels.*.name, 'extended-ci')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Check out the scylla-bench repository
+        uses: actions/checkout@v2
+        with:
+          repository: scylladb/scylla-bench
+          path: scylla-bench
+
+      - name: Checkout GoCQL PR Repository
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          path: gocql
+
+      - name: Build and push Scylla-bench Docker Image with gocql from PR
+        run: |
+          cd scylla-bench
+          GOCQL_REPO="github.com/${{ github.event.pull_request.head.repo.full_name }}" GOCQL_VERSION="${{ github.event.pull_request.head.ref }}" make build-with-custom-gocql-version
+          DOCKER_IMAGE_TAG="scylladb/gocql-extended-ci:scylla-bench-${{ github.event.pull_request.head.sha }}" DOCKER_IMAGE_LABELS="com.scylladb.gocql-version=${{ github.event.pull_request.head.sha }}" make build-sct-docker-image
+          docker push "scylladb/gocql-extended-ci:scylla-bench-${{ github.event.pull_request.head.sha }}"
+
+      - name: Start Jenkins job
+        uses: sylwiaszunejko/jenkins_client@main
+        with:
+          jenkins_job_name: 'scylla-drivers/gocql/extended-ci/longevity-large-partition-asymmetric-cluster-3h-test'
+          jenkins_job_parameters: '{"email_recipients": "sylwia.szunejko@scylladb.com", "scylla_version": "6.1.1", "extra_environment_variables": "SCT_STRESS_IMAGE.scylla-bench=scylladb/gocql-extended-ci:scylla-bench-${{ github.event.pull_request.head.sha }}"}'
+          jenkins_base_url: 'https://jenkins.scylladb.com/'
+          jenkins_user: sylwiaszunejko
```

Comment:
@yaronkaikov do we have a jenkins user we can for the CI?

Reply:
I think you can use `scylladb-promoter` but we should probably also set the `token` as `GITHUB_SECRETS`

Reply:
Let's have those creds at org level, we'll need them in more projects that would follow

Reply:
Done, should use `JENKINS_USERNAME` and `JENKINS_TOKEN`, it's organization level, for now only for the `scylladb/gocql` repo. Let's make sure it's working then we can add additional repos

Reply:
For some reason triggering job does not work with the credentials provided by @yaronkaikov  https://github.com/scylladb/gocql/actions/runs/11120234541/job/30896983496?pr=263 my guess is that JENKINS_TOKEN secret on the org level has the same name as token we used for my username, @dkropachev could you maybe remove this repo level token?

Reply:
Done